near-plugins/tests/contracts/access_controllable/src/lib.rs

```diff
@@ -49,3 +71,32 @@ impl StatusMessage {
     // }
     // ```
 }
+
+/// Exposing internal methods to facilitate integration testing.
+#[near_bindgen]
+impl StatusMessage {
+    #[private]
+    pub fn acl_add_super_admin_unchecked(&mut self, account_id: AccountId) -> bool {
+        self.__acl.add_super_admin_unchecked(&account_id)
+    }
+
+    #[private]
+    pub fn acl_revoke_super_admin_unchecked(&mut self, account_id: AccountId) -> bool {
+        self.__acl.revoke_super_admin_unchecked(&account_id)
+    }
+
+    #[private]
+    pub fn acl_add_admin_unchecked(&mut self, role: Role, account_id: AccountId) -> bool {
+        self.__acl.add_admin_unchecked(role, &account_id)
+    }
+
+    #[private]
+    pub fn acl_revoke_admin_unchecked(&mut self, role: Role, account_id: AccountId) -> bool {
+        self.__acl.revoke_admin_unchecked(role, &account_id)
+    }
+
+    #[private]
+    pub fn acl_grant_role_unchecked(&mut self, role: Role, account_id: AccountId) -> bool {
+        self.__acl.grant_role_unchecked(role, &account_id)
+    }
+}
```

Comment:
Let's add this behind [integration-test] feature flag, similar as it is used in the engine today.

Reply:
https://github.com/aurora-is-near/near-plugins/blob/b2e1d8369059bee4b0e31cb92b965a59a8a2717f/near-plugins/tests/access_controllable.rs#L25

The contract containing these methods is compiled and deployed via [`workspaces::compile_project`](https://docs.rs/workspaces/latest/workspaces/fn.compile_project.html) which has only one parameter: `project_path: &str`. AFAIK this prevents us from working with features in this contract, since we'd have to enable required features by default (see below).

The following works, though if the `integration-test` feature needs to be enabled by default, I'm not sure if it makes sense to have it?

```
[features]
default = ["integration-test"]
integration-test = []
```

---

It might be worth a feature request for `compile_project` to have an additional parameter `&[&str]` that specifies the features to be enabled.